test_people.py

```diff
@@ -9,6 +9,14 @@ def test_oliver():
     assert oliver.hobbies == []
 
 
+def test_squanchy():
+    squanchy = setup_people()['squanchy_smith']
+    assert squanchy.first_name == 'squanchy'
+    assert squanchy.age == 21
+    assert squanchy.hobbies == ['squanching','running']
```

Comment:
according to the Git Actions CI/CD pytest.
```
./test_people.py:16:45: E231 missing whitespace after ','
./test_people.py:20:1: E303 too many blank lines (3)
./people.py:29:60: E231 missing whitespace after ','
./people.py:33:1: E302 expected 2 blank lines, found 1
2     E231 missing whitespace after ','
1     E302 expected 2 blank lines, found 1
1     E303 too many blank lines (3)
```
test_squancy should have a space after the comma on line 16 etc.

Please make sure that pycodestyle is clean before commit and check the GitHub actions flake8 log is clean (see video on Canvas).

Reply:
fixed in 751fc96

is this ok?